src/util/adb.js

```diff
@@ -93,6 +93,8 @@ export default class ADBUtils {
       'pm',
       'list',
       'packages',
+      '--user',
+      '0',
```

Comment:
While this may work for your specific setup, I think that a more general fix is to use `adb shell am get-current-user` to get the actual user that's currently running on the phone, and use that instead of hardcoding 0. Can you give that a try?

Reply:
Unfortunately you are right that would only work "on most devices" per [Android docs](https://android.googlesource.com/platform/frameworks/base/+/master/core/java/android/os/Users.md).
I can give it a try, though not immediately due to some other hustle.
That seem to be an ADB bug to some extent. Disabled user does not seem to be the best default on their end.

Reply:
@lm1 Have you had a chance to revisit this PR and test it?

Reply:
Updated and tested.